api/api.go

```diff
@@ -61,32 +70,32 @@ func (h *httpHandler) handleRequest(w http.ResponseWriter, req *http.Request) {
 func (h *httpHandler) initRouter() {
 	m := map[string]map[string]processor{
 		"GET": {
-			"/networks":                                                                   procGetNetworks,
-			"/networks/name/{" + urlNwName + ":.*}":                                       procGetNetwork,
-			"/networks/id/{" + urlNwID + ":.*}":                                           procGetNetwork,
-			"/networks/name/{" + urlNwName + ":.*}/endpoints/":                            procGetEndpoints,
-			"/networks/id/{" + urlNwID + ":.*}/endpoints/":                                procGetEndpoints,
-			"/networks/name/{" + urlNwName + ":.*}/endpoints/name/{" + urlEpName + ":.*}": procGetEndpoint,
-			"/networks/id/{" + urlNwID + ":.*}/endpoints/name/{" + urlEpName + ":.*}":     procGetEndpoint,
-			"/networks/name/{" + urlNwName + ":.*}/endpoints/id/{" + urlEpID + ":.*}":     procGetEndpoint,
-			"/networks/id/{" + urlNwID + ":.*}/endpoints/id/{" + urlEpID + ":.*}":         procGetEndpoint,
+			"/networks":                                              procGetNetworks,
+			"/networks/name/" + nwName:                               procGetNetwork,
+			"/networks/id/" + nwID:                                   procGetNetwork,
+			"/networks/name/" + nwName + "/endpoints":                procGetEndpoints,
+			"/networks/id/" + nwID + "/endpoints":                    procGetEndpoints,
+			"/networks/name/" + nwName + "/endpoints/name/" + epName: procGetEndpoint,
+			"/networks/id/" + nwID + "/endpoints/name/" + epName:     procGetEndpoint,
+			"/networks/name/" + nwName + "/endpoints/id/" + epID:     procGetEndpoint,
+			"/networks/id/" + nwID + "/endpoints/id/" + epID:         procGetEndpoint,
 		},
 		"POST": {
```

Comment:
Thanks. Also, can you please change `GET` and `DELETE` URLs to remove `/id/` and `/name/` from the URI ?
It would be great to have the identification of `id` or `name` in the backend.